docs/default-firebase-data.json

```diff
@@ -394,67 +491,392 @@
       "order": 0
     }
   },
-  "team": [{
-      "members": [{
-        "name": "Oleh Zasadnyy",
-        "photo": "/images/people/oleh_zasadnyy.jpg",
-        "photoUrl": "https://storage.googleapis.com/hoverboard-experimental.appspot.com/images/people/optim/oleh_zasadnyy.jpg",
-        "socials": [{
-            "icon": "facebook",
-            "link": "https://www.facebook.com/oleh.zasadnyy",
-            "name": "Facebook"
-          },
-          {
-            "icon": "twitter",
-            "link": "https://twitter.com/ozasadnyy",
-            "name": "Twitter"
-          },
-          {
-            "icon": "website",
-            "link": "http://o.zasadnyy.com/",
-            "name": "Blog"
-          }
-        ],
-        "order": 0,
-        "title": "Website and sponsors"
-      }],
-      "title": "Core Team"
+  "team": [
+    {
+      "members": [
+        {
+          "name": "Natalie Pis",
+          "photo": "/images/team/natalie.jpg",
+          "photoUrl": "https://devfest-berlin-2019-testing-a.firebaseapp.com/images/team/natalie.jpg",
+          "socials": [
+            {
+              "icon": "twitter",
+              "link": "https://twitter.com/NataliePis",
+              "name": "Twitter"
+            }
+          ],
+          "order": 0,
+          "title": "Organizer"
+        },
+        {
+          "name": "Ole Bulbuk",
+          "photo": "/images/team/ole_bulbuk.jpg",
+          "photoUrl": "https://devfest-berlin-2019-testing-a.firebaseapp.com/images/team/ole_bulbuk.jpg",
+          "socials": [
+            {
+              "icon": "twitter",
+              "link": "https://twitter.com/ole108",
+              "name": "Twitter"
+            },
+            {
+              "icon": "github",
+              "link": "https://github.com/ole108",
+              "name": "GitHub"
+            }
+          ],
+          "order": 1,
+          "title": ""
+        },
+        {
+          "name": "Anderson Queiroz",
+          "photo": "/images/team/anderson_queiroz.jpg",
+          "photoUrl": "https://devfest-berlin-2019-testing-a.firebaseapp.com/images/team/anderson_queiroz.jpg",
+          "socials": [
+            {
+              "icon": "twitter",
+              "link": "https://twitter.com/AinSoph",
+              "name": "Twitter"
+            }
+          ],
+          "order": 2,
+          "title": ""
+        },
+        {
+          "name": "Eden",
+          "photo": "",
+          "photoUrl": "https://devfest-berlin-2019-testing-a.firebaseapp.com/images/team/",
```

Comment:
That is a wrong URL. If we don't want it, maybe we can skip it? 🙈

Reply:
yea we can skip it for sure 👍